src/main/java/nextstep/model/Players.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class Players {
+    private final List<String> names;
+
+    public Players(String[] names) {
+        this.names = new ArrayList<>();
+        Optional<String> first = Arrays.stream(names)
```

Comment:
```suggestion
        Arrays.stream(names).anyMatch(s -> s.length() > 5);
```
`anyMatch`를 한번 활용해보시는건 어떠실까요?? 🤔